src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -5643,3 +5643,25 @@ TaskListHighestTaskId(List *taskList)
 
 	return highestTaskId;
 }
+
+
+/*
+ * RemoveCoordinatorPlacementFromList walks over shard placements in the given list,
+ * and removes all that are placed in the coordinator.
+ *
+ * The list is assumed to be in increasing group id values, and a coordinator
+ * placement is assumed to be in the first K (K>=0) cells.
+ */
+List *
+RemoveCoordinatorPlacementFromList(List *placementList)
```

Comment:
Is this different than `RemoveCoordinatorPlacement` in router planner?

Reply:
Actually it's not. Thanks for pointing it out. I removed this function